web/src/pages/Profile/JurorCard/Header.tsx

```
@@ -10,7 +10,9 @@ import XIcon from "svgs/socialmedia/x.svg";
 import HowItWorks from "components/HowItWorks";
 import JurorLevels from "components/Popup/MiniGuides/JurorLevels";
 import { ExternalLink } from "components/ExternalLink";
+import HowItWorks from "components/HowItWorks";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'HowItWorks'. Consider to delete it or rename it.

'HowItWorks' is defined here:


(lint/suspicious/noRedeclare)